tools/rust/crane.nix

```diff
@@ -363,6 +363,18 @@
                   " -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort --target ${buildStdTarget}"
                 );
               RUSTFLAGS = rustflags;
+
+              preBuild = ''
+                echo "cargoVendorDir: ${crateAttrs.cargoVendorDir}"
+                echo "rustToolchain: ${cargoBuildRustToolchain'}"
+
+                # find ${crateAttrs.cargoVendorDir} -maxdepth 1 -xtype d | grep -v '^${crateAttrs.cargoVendorDir}$' | sed -E 's@(.+)@ --remap-path-prefix=\1=/@g'
```

Comment:
Remove this?

Reply:
i kept it bc it was very useful for debugging, to see what's happening internally (especially if crane changes their vendoring logic)